src/AspNetCoreModuleV2/AspNetCore/src/dllmain.cpp

```diff
@@ -19,7 +19,8 @@ DWORD               g_dwActiveServerProcesses = 0;
 SRWLOCK             g_srwLock;
 DWORD               g_dwDebugFlags = 0;
 PCSTR               g_szDebugLabel = "ASPNET_CORE_MODULE";
-PCWSTR              g_pwzAspnetcoreRequestHandlerName = L"aspnetcorerh.dll";
+PCWSTR              g_pwzAspnetcoreInProcessRequestHandlerName = L"aspnetcorev2_inprocess.dll";
```

Comment:
I'd rather have a longer unambiguous name. Any reason you want a shorter name?

Reply:
Windows native dll traditionally has short name so that it can fix MAX_PATH for most cases and better perf when doing file search based on string comparison.

Reply:
It's only 10 character longer than aspnetcorev2.dll

Reply:
This dll is also loaded from the shared framework/ bin folder, so if this dll couldn't be loaded, no other dotnet core dlls could be loaded either. If we see customer issues or feel strongly otherwise, we can revert/modify the name.